decidim-proposals/app/controllers/decidim/proposals/admin/valuation_assignments_controller.rb

```diff
@@ -13,6 +13,10 @@ def create
             on(:ok) do |_proposal|
               flash[:notice] = I18n.t("valuation_assignments.create.success", scope: "decidim.proposals.admin")
               redirect_to EngineRouter.admin_proxy(current_component).root_path
+
+              Decidim::Proposals::Admin::ProposalsValuatorMailer.notify_proposals_valuator(
+                @form.valuator_user, current_user, @form.proposals, _proposal
+              ).deliver_now
```

Comment:
- We should move this to the class `Admin::AssignProposalsToValuator` (in app/commands).
-  better use `deliver_later` too